libs/ledger-live-common/src/hw/index.ts

```diff
@@ -1,8 +1,8 @@
 import { EMPTY, merge } from "rxjs";
 import type { Observable } from "rxjs";
 import { catchError } from "rxjs/operators";
+import type { DeviceModel } from "@ledgerhq/devices";
```

Comment:
I know we have to go through all our types on the device parts because it's a bit of a mess :(

But we are not using the new type lib `types-devices` anymore ? `DeviceModel` does exist on `libs/ledgerjs/packages/types-devices/src/index.ts`